BoxBrowseSDK/BoxBrowseSDK/Categories/NSBundle+BOXBrowseSDKAdditions.m

```diff
@@ -13,14 +13,9 @@ @implementation NSBundle (BOXBrowseSDKAdditions)
 
 + (NSBundle *)boxBrowseSDKResourcesBundle
 {
-    static NSBundle *browseSDKResourceBundle = nil;
-    static dispatch_once_t predicate;
-    dispatch_once(&predicate, ^{
-        NSBundle *browseSDKBundle = [NSBundle bundleForClass:[BOXBrowseSDKResourceLocator class]];
-        NSString *browseSDKResourceBundlePath = [browseSDKBundle.bundlePath stringByAppendingPathComponent:@"BoxBrowseSDKResources.bundle"];
-        browseSDKResourceBundle = [NSBundle bundleWithPath:browseSDKResourceBundlePath];
-    });
-
+    NSBundle *browseSDKBundle = [NSBundle bundleForClass:[BOXBrowseSDKResourceLocator class]];
```

Comment:
What about caching it and having a dedicated thread for accessing it? Not sure if it's worth it, but I am concerned about performance.